src/commonMain/kotlin/OpenTest4kH.kt

```diff
@@ -140,7 +160,7 @@ expect open class AssertionFailedError : AssertionError {
      * converted to the empty `String`
      * @param cause the cause of the failure
      */
-    constructor(message: String? = null, cause: Throwable? = null)
+    constructor(message: String?, cause: Throwable?)
```

Comment:
Default parameters of the constructors are non available since 1.9.20. 
You whether need to remove them or stop using `actual typealias` construction:
```kotlin
actual typealias AssertionFailedError = org.opentest4j.AssertionFailedError
```
See: https://youtrack.jetbrains.com/issue/KT-57614